R/MixedModelsCommon.R

```diff
@@ -1168,6 +1155,11 @@
       plots$setError(p$message)
       return()
     }
+
+    if (options$plotsGeom == "geom_violin" && (length(options$plotsAgregatedOver) == 1 && length(unique(dataset[, .v(options$plotsAgregatedOver)])) < 3)) {
+      plots$setError(gettext("Violin geom requires that the random effects grouping factors has at least 3 levels."))
```

Comment:
The user would not understand what a violin geom is.

Reply:
It's called violin geom in the menu

![image](https://user-images.githubusercontent.com/38475991/121767703-40d30b00-cb5a-11eb-9eec-447c7684fc1f.png)

Reply:
That still doesn't make sense to the users.

Reply:
Not sure what you want from me. Change the error message, rename the dropdown, or both? Then we would need a new, different name than ``Background geom`` to describe the geom shape which depicts the background data.